cmd/consumer/main.go

```diff
@@ -143,7 +143,7 @@ func appendToFile(filename, text string) error {
 func (rp *RecordProcessor) Shutdown(checkpointer kcl.Checkpointer, reason string) error {
 	if reason == "TERMINATE" {
 		fmt.Fprintf(os.Stderr, "Was told to terminate, will attempt to checkpoint.\n")
-		rp.firehoseWriter.FlushAll()
+		rp.firehoseWriter.Shutdown()
 		rp.checkpoint(checkpointer, "", 0)
```

Comment:
Is there an error returned from `FlushAll`? Perhaps we shouldn't checkpoint if the flush was not successful...